### DISCUSSION TITLE
Where are the binaries?

### DISCUSSION THREAD

**Discussion by KenWi11iams** [pmd/pmd#4732] [Q&A] [answered]
The instructions tell me to "Download the latest binary zip from the [releases](https://github.com/pmd/pmd/releases/latest) and extract it somewhere."  When I go to the webpage, I cannot find any binaries.  If I do the usual github download a zip, all I get is what appears to be the source code and documentation.  Is a compiled version available as the documentation states?

**Comment by oowekyala** [comment, marked as answer]
In the page you linked there is a link to [pmd-dist-7.0.0-rc4-bin.zip](https://github.com/pmd/pmd/releases/download/pmd_releases%2F7.0.0-rc4/pmd-dist-7.0.0-rc4-bin.zip) if you scroll down to the end of the page (section "Assets").
